### DISCUSSION TITLE
How to enable sha3()?

### DISCUSSION THREAD

**Discussion by allchannelson** [sqlitebrowser/sqlitebrowser#3262] [Q&A] [answered]
Hello. I'm using SQLite version 3.40.1 and have also loaded the respective sqlite3.dll in the root folder of DB Browser, but the sha3() function gives me an error and I'm not sure why.

If I use the CLI version, sha3() works fine.

**Comment by justinclift** [comment]
Interesting.  What's the error message it gives you?

Oh, as a thought... when you say you've "loaded the respective sqlite3.dll in the root folder", are you meaning you just gave a new/updated .dll instead of using a newer (eg nightly) version of our application?

If that's the case, then it's probably a good idea to try the [latest nightly build instead](https://nightlies.sqlitebrowser.org/latest/), as that has the newest SQLite version already built in.  Should mean things "just work as intended" without any weirdness.

**Reply by bluebrown** [reply]
i am also missing it and I cant find documentation for it either. It doesnt seem to actually exist. There are some forum posts and stuff where its mentioned, but I dont find docs and I dont have it in my sqlite3 installation.

**Reply by FriedrichFroebel** [reply, marked as answer]
`shatree` is an optional extension to SQLite3, thus it has to be compiled by you and loaded as an extension into DB4S: https://www.sqlite.org/src/dir?ci=trunk&name=ext/misc

**Reply by justinclift** [reply]
@lucydodo We might want to add sha3 to the list of included extensions at some point too. :smile:

**Reply by lucydodo** [reply]
@justinclift Okay, great, I'll put it on my to-do list. :)
